spec/support/spree/config_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module ConfigHelper
+    def with_spree_config(hash)
+      stored = config_slice(hash.keys)
+      Config.set(hash)
+      yield
+      Config.set(stored)
```

Comment:
Yeah cool. 👍 I am fairly sure that any unmentioned config keys are left untouched, but I'll double check.